src/api/market.ts

```diff
@@ -1,28 +1,29 @@
-import axios from 'axios';
 import { Axios } from './Axios';
-import { NavigateFunction } from 'react-router-dom';
 
-export const getProducts = async (
+export const getProducts = async (token: string | null, categoryId: number) => {
```

Comment:
token이 string | null 이라면 null 일 때의 처리를 좀 해줘야 할듯해요
지금 코드대로라면 null 일 때 서버로 빈 토큰이 날아가버리는 이슈가 있어서, null 이면 에러를 throw 하거나 하는 방향으로 코드 짜보는 것도 좋을듯!

Reply:
에러 throw하게 고쳐봤는데 봐주시면 감사하겠습니다 ,, (•͈ᴗ•͈)

Reply:
흠.. 근데 함수의 선언부를 아래처럼 하고 token 이 없을 땐 아예 이 함수를 호출하지 않도록 만드는게 낫지 않을까 싶어요!
```suggestion
export const getProducts = async (token: string, categoryId: number) => {
```